src/marlin_stubs/pause/pause.cpp

```diff
@@ -579,6 +589,13 @@ void Pause::unpark_nozzle_and_notify() {
     const float Z_len = current_position.z - resume_pos.z; // sign does not matter, does not check Z max val (unlike park_nozzle_and_notify)
     const float XY_len = begin_pos - end_pos;              // sign does not matter
 
+    //home the axis if it is not homed
```

Comment:
Just comment:
Unparking is after parking, so axis should be already homed

Reply:
Beacuse we can move to parking point in one axis only and the resume point could need to move in the other axis.